CONTRIBUTING.md

```diff
@@ -1,10 +1,22 @@
 # Contributing
 
+## Introduction
+
+This book is a catalogue of Rust programming techniques, (anti-)patterns, and idioms and other explanations.
```

Comment:
```suggestion
This book is a catalogue of Rust programming techniques, (anti-)patterns, idioms and other explanations.
```

I suggest having just one `and` here, by replacing the `, and idioms` with just `, idioms`.

I'm referencing this: https://english.stackexchange.com/a/187872, which states the following:

`
There is nothing that would stop you from using 'and' multiple times, but if you are to list things, using a comma is better. 
Example: "I like to play football, ride my bike, play guitar and hang with friends." 
Instead of "I like to play football and ride my bike and play guitar and hang with friends".
`